docs/sql-ref-syntax-dml-insert-overwrite-table.md

```diff
@@ -41,7 +41,7 @@ INSERT OVERWRITE [ TABLE ] table_identifier [ partition_spec [ IF NOT EXISTS ] ]
 * **partition_spec**
 
     An optional parameter that specifies a comma separated list of key and value pairs
-    for partitions.
+    for partitions. Here we support use type constructed value as partition col value.
```

Comment:
I have the same comment with `sql-ref-syntax-dml-insert-into.md `

Reply:
Done